### PR TITLE
21594 - Refund requested badge on short name management list

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/21594

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/utils/enums.py
Comment (ochiu) [inline pay-api/src/pay_api/utils/enums.py:358]: I mentioned this as well on PR #1719, the EFT Refund flow has it's own flow and states for pending, approval, rejection. There should be a new set of statuses. When we create a refund we are also using the EFTCreditInvoiceStatus enum instead.

File: pay-api/src/pay_api/services/eft_short_name_summaries.py
Comment (ochiu) [inline pay-api/src/pay_api/services/eft_short_name_summaries.py:134]: Can we have multiple pending short name refunds? It would duplicate rows in that case?

Comment (Jxio) [reply pay-api/src/pay_api/services/eft_short_name_summaries.py:134]: it will, I added distinct to avoid duplicate rows

File: pay-api/src/pay_api/services/eft_short_name_summaries.py
Comment (seeker25) [inline pay-api/src/pay_api/services/eft_short_name_summaries.py:83]: do we need this?

File: pay-api/src/pay_api/services/eft_short_name_summaries.py
Comment (seeker25) [inline pay-api/src/pay_api/services/eft_short_name_summaries.py:82]: doesn't look like a count to me

File: pay-api/src/pay_api/services/eft_short_name_summaries.py
Comment (seeker25) [reply pay-api/src/pay_api/services/eft_short_name_summaries.py:134]: where is the distinct?

File: pay-api/tests/unit/api/test_eft_short_names.py
Comment (seeker25) [inline pay-api/tests/unit/api/test_eft_short_names.py:287]: `assert shortname_refund is None or result_dict['refundStatus'] == shortname_refund.status`

File: pay-api/tests/unit/api/test_eft_short_names.py
Comment (seeker25) [inline pay-api/tests/unit/api/test_eft_short_names.py:551]: SUP!

File: pay-api/tests/unit/api/test_eft_short_names.py
Comment (seeker25) [inline pay-api/tests/unit/api/test_eft_short_names.py:287]: `assert result_dict['refundStatus'] == (shortname_refund.status if shortname_refund is not None else None)`

File: pay-api/src/pay_api/services/eft_short_name_summaries.py
Comment (Jxio) [reply pay-api/src/pay_api/services/eft_short_name_summaries.py:133]: just pushed

File: pay-api/src/pay_api/services/eft_short_name_summaries.py
Comment (Jxio) [reply pay-api/src/pay_api/services/eft_short_name_summaries.py]: removed

**Review by seeker25 — APPROVED** [review]
Ok to merge as long as Ody is ok with it

**Review by ochiu — APPROVED** [review]

**Pull request merged**
